### PR TITLE
[Feature] Integrate Aleo Javascript SDK into aleo.tools

### LINKED ISSUE #713: [Feature] Integrate SDK into Aleo.tools
## 🚀 Feature

The Aleo SDK has been updated to include program execution, management, and deployment. It should be used to power the aleo.tools website.

### PR DESCRIPTION
**Pull request by iamalwaysuncomfortable**
## Motivation

The Aleo SDK was recently developed to provide a full fledged web development toolkit to web app developers. This PR merges the Javascript SDK into Aleo.tools

Preview: https://improved-loading--aleo-tools-3481ab.netlify.app/develop

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -4,16 +4,16 @@
...
         "build": "webpack --config webpack.config.js",
         "build:vite": "vite build",
+        "dev:webpack": "webpack-dev-server",
         "lint": "eslint src --ext js,jsx --report-unused-disable-directives --max-warnings 0 && prettier . --check",
         "pretty": "prettier . --write",
         "preview": "vite preview"
     },
     "dependencies": {
-        "@aleohq/wasm": "file:../wasm/pkg-parallel",
+        "@aleohq/sdk": "0.5.5",
```
Comment (iamalwaysuncomfortable) [inline website/package.json]: This a subtle but key part of the PR

File: website/src/tabs/develop/Deploy.jsx [hunk 2]
```diff
@@ -34,23 +34,11 @@ export const Deploy = () => {
...
-                        setFeeLoading(false);
-                        setLoading(false);
-                        setDeploymentError(null);
-                        setTransactionID(response.data);
-                    });
+                let transactionId = ev.data.deployTransaction;
+                setFeeLoading(false);
+                setLoading(false);
+                setDeploymentError(null);
+                setTransactionID(transactionId);
```
Comment (iamalwaysuncomfortable) [inline website/src/tabs/develop/Deploy.jsx:41]: In each file the SDK now encapsulates the network calls necessary to post a transaction

File: website/src/workers/worker.js [hunk 3]
```diff
@@ -1,97 +1,20 @@
-import init, * as aleo from "@aleohq/wasm";
-import {
-    FEE_PROVER_URL,
```
Comment (iamalwaysuncomfortable) [inline website/src/workers/worker.js:3]: The SDK also encapsulates the proving key downloads

File: website/src/workers/worker.js [hunk 4]
```diff
@@ -101,37 +24,50 @@ self.addEventListener("message", (ev) => {
...
                     lastLocalProgram = localProgram;
                 }
 
-                let response = aleoProgramManager.executeFunctionOffline(
-                    aleo.PrivateKey.from_string(privateKey),
+                // Pass the cache key to the execute function
+                const keyParams = new aleo.AleoKeyProviderParams({"cacheKey": cacheKey});
+
+                // Execute the function locally
+                let response = await programManager.executeOffline(
```
Comment (iamalwaysuncomfortable) [inline website/src/workers/worker.js:48]: The program manager is now directly in the SDK

**Review by onetrickwolf — APPROVED** [review]

**Pull request merged**